notebooks/05_operationalize/pipeline-demo/aml-pipeline-use-datebricks-as-compute-target.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>The other notebooks are in the root of `05_operationalize`, probably it would be nice to keep with that pattern to reduce a complex folder structure</p>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>